Formula/gnu-tar.rb

```diff
@@ -11,6 +11,13 @@ class GnuTar < Formula
     sha256 "1a559b78e6f1a6594b18a9ba2aa2e9828af2736aacc4aec07911fe7638e80e68" => :el_capitan
   end
 
+  head do
+    depends_on "automake"
+    depends_on "autoconf"
+    depends_on "gettext"
+    url "git://git.savannah.gnu.org/tar.git"
```

Comment:
url should be listed above the dependencies and there should be an empty line between url and depends_on.

Reply:
fixed, thanks